src/main/resources/config/liquibase/changelog/20220903100502_changelog.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.8.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="ge65cer" id="20220903100502-1">
+        <comment>Fill the learning_goal_exercise table with relations currently stored in the learning_goal_lecture_unit table.</comment>
+        <sql>
+            INSERT INTO learning_goal_exercise (learning_goal_id, exercise_id)
+            SELECT lg.learning_goal_id, lu.exercise_id FROM learning_goal_lecture_unit lg
+            LEFT JOIN lecture_unit lu ON lu.id = lg.lecture_unit_id
```

Comment:
❓Question: Not a blocker but just out of interest why a left join is used here. There should always be a lecture unit no?

Reply:
Yes, due to our constraints there will always be a lecture unit for the id in the relations table. To be 100% percent safe, I can change this to an inner join.